### PR TITLE
[python]challenge_[6](Pending)

### PR DESCRIPTION
**Pull request by MJUIUC**
Here's my Solution for challenge 6.

### CODE REVIEW

File: challenge_6/python/mjuiuc/test.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import unittest
+import time
+from Solution import solution
+
```
Comment (halogenandtoast) [inline challenge_6/python/mjuiuc/test.py]: Is this much whitespace typical in Python? I would assume 1 newline would suffice. Thoughts on removing the extra newlines.

File: challenge_6/python/mjuiuc/Solution.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
+#	Writing solution for this range problem
+
+class solution(object):
```
Comment (halogenandtoast) [inline challenge_6/python/mjuiuc/Solution.py]: Can this just be `class Solution:`?

File: challenge_6/python/mjuiuc/Solution.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+	def compRange(self,nums):
```
Comment (halogenandtoast) [inline challenge_6/python/mjuiuc/Solution.py]: I believe the PEP 8 styleguide wants a space after commas.

File: challenge_6/python/mjuiuc/test.py [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+if __name__ == '__main__':
+	unittest.main()
```
Comment (halogenandtoast) [inline challenge_6/python/mjuiuc/test.py]: Missing newline.

File: challenge_6/python/mjuiuc/test.py [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+class OutcomesTest(unittest.TestCase):
+
+	def test1(self):
+		tlist = [1,2,3,4,5,8,9,10]
+		case1 = solution()
+		t1 = time.time()
+		out = case1.compRange(tlist)
+		t2 = time.time()
+		print "Runtime for case1: " + str(t2-t1) + " seconds"
```
Comment (halogenandtoast) [inline challenge_6/python/mjuiuc/test.py]: I believe the pep 8 styleguide implies that spaces should go around all binary operators like " - " and " % "

File: challenge_6/python/mjuiuc/Solution.py (hunk 3 above)
Comment (MJUIUC) [inline challenge_6/python/mjuiuc/Solution.py]: I didn't know what PEP was until I read this comment.

File: challenge_6/python/mjuiuc/Solution.py (hunk 2 above)
Comment (MJUIUC) [inline challenge_6/python/mjuiuc/Solution.py]: I suppose, but I don't think that's a big deal.

**Pull request merged**